xbmc/addons/ImageResource.cpp

```diff
@@ -18,9 +18,9 @@ namespace ADDON
 {
 
 CImageResource::CImageResource(const AddonInfoPtr& addonInfo)
-  : CResource(addonInfo, AddonType::RESOURCE_IMAGES)
+  : CResource(addonInfo, AddonType::RESOURCE_IMAGES),
+    m_type(Type(AddonType::RESOURCE_IMAGES)->GetValue("@type").asString())
```

Comment:
also this one as below, what kind of warning do the compiler for these two cases?

Reply:
Yes, please include the warnings in the commit descriptions @tobbi

Reply:
Like I said in the other PR, reason for the change is performance, not readability. (init members with desired value in init list)